Mod Menu Helper/assets/modmenu/lang/de_de.json

```diff
@@ -0,0 +1,43 @@
+    "modmenu.summaryTranslation.cem": "Einstellbare Enitäten, wie bei Optifine (Alpha)\n§9✎§8 Optifine wird unterstützt, behobenes Modell",
+    "modmenu.summaryTranslation.dcch": "Chatverlauf hinweg über Welten und Servern behalten\n§8✎ (keine Optionen)",
+    "modmenu.summaryTranslation.dynamicfps": "FPS im Hintergrund verringern\n§9✎§8 Mod ein-/ausstellen, bei Maushover zurücksetzen, ...",
+    "modmenu.summaryTranslation.enhancedblockentities": "Optimiert Truhen, Schilder, Betten usw.\n§9✎§8 zu optimierende Blöcke, Methode",
+    "modmenu.summaryTranslation.entityculling": "Verhindert das Rendern von Mobs hinter Mauern\n§8✎ (nur fortgeschrittene Optionen)",
+    "modmenu.summaryTranslation.fabrishot": "HD Screenshots mit F9\n§9✎§8 Überschreibe F2, Auflösung, Taste, ...",
+    "modmenu.summaryTranslation.fastchest": "§c*§r Optimierte Truhen\n§9✎§8 Mod ein-/ausstellen",
+    "modmenu.summaryTranslation.ferritecore": "Verringert Arbeitsspeicherauslastung\n§8✎ nur fortgeschrittene Optionen)",
+    "modmenu.summaryTranslation.hydrogen": "Verringert (auch) Arbeitsspeicherauslastung\n§8✎ keine Optionen)",
+    "modmenu.summaryTranslation.iris": "Optifine-Shader Unterstützung\n§9✎§8 Mod ein-/ausstellen, Shader auswählen, Taste, ...",
+    "modmenu.summaryTranslation.itemmodelfix": "Fix transparent lines in held items\n§9✎§8 Method",
+    "modmenu.summaryTranslation.lambdabettergrass": "\"Besseres Graß\"-Einstellung, wie bei Optifine\n§9✎§8 Qualität, Schnee, Ressourcenpaket",
+    "modmenu.summaryTranslation.lambdynlights": "\"Dynamische Lichter\"-Einstellung, wie bei Optifine\n§9✎§8 Qualität, Enitäteneinstellungen, ...",
+    "modmenu.summaryTranslation.languagereload": "Direktes Wechseln der Sprache, suchbar\n§9✎§8 F3+J, um eine Sprache neuzuladen",
+    "modmenu.summaryTranslation.lazydfu": "Schnellerer Spielstart\n§8✎ (keine Optionen)",
+    "modmenu.summaryTranslation.lithium": "Optimiert Spielphysik und Chunks\n§8✎ (nur fortgeschrittene Optionen)",
+    "modmenu.summaryTranslation.modmenu": "Dieses Modmenü\n§9✎§8 Knopfeinstellungen, Filtern, Sortieren, ...",
+    "modmenu.summaryTranslation.morechathistory": "Erhöht das Charverlauf-Limit\n§8✎ (keine Optionen)",
+    "modmenu.summaryTranslation.nofade": "Entfernt Lade/Verblass-Animationen\n§8✎ (keine Optionen)",
+    "modmenu.summaryTranslation.no-telemetry": "Deaktiviert das Senden von Diagnostikdaten\n§8✎ (keine Optionen)",
+    "modmenu.summaryTranslation.notenoughcrashes": "Erlaubt das weitere Spielen nach einem Absturz\n§8✎ (nur fortgeschrittene Optionen)",
+    "modmenu.summaryTranslation.phosphor": "Optimiert Licht, schnelleres Laden von Chunks\n§8✎ (nur fortgeschrittene Optionen)",
+    "modmenu.summaryTranslation.smoothboot": "Verhindert Spielabstürze\n§8✎ (nur fortgeschrittene Optionen)",
+    "modmenu.summaryTranslation.sodium": "Optimiert das Rendern, mehr FPS\n§9✎§8 In den Videoeinstellungen: Optimierungen",
```

Comment:
"Videoeinstellungen" should be "Grafikeinstellungen" as that is how it's translated in German